BookingsAPI/Bookings.API/Mappings/SuitabilityAnswersListToResponseMapper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Bookings.Api.Contract.Responses;
+using Bookings.Domain.Participants;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Bookings.API.Mappings
+{
+    public class SuitabilityAnswersListToResponseMapper
+    {
+        public List<ParticipantSuitabilityAnswerResponse> MapParticipantSuitabilityAnswerResponses(IEnumerable<Participant> participants)
+        {
+            var mapped = participants.Select(MapParticipantResponse);
+            return mapped.OrderByDescending(x => x.UpdatedAt).ToList();
+        }
+
+        public ParticipantSuitabilityAnswerResponse MapParticipantResponse(Participant participant)
+        {
+            var @case = participant.Hearing.GetCases().FirstOrDefault();
+            var @representative = participant as Representative;
+            var answerMapper = new SuitabilityAnswerToResponseMapper();
```

Comment:
Injected, can create a mapper interface ?

Reply:
I don't think we need at the moment DI complexity here, principal YAGNI says not to worry so much over it now 'cos you probably won't need to spend that extra effort. Get it done, come back to it if you really need to.
 We can refactor all mapping if we need to introduce Interface.